frontend/src/metabase/ui/components/feedback/Loader/Loader.tsx

```diff
@@ -0,0 +1,14 @@
+import { Loader as MantineLoader } from "@mantine/core";
+import type { LoaderProps } from "@mantine/core";
+
+const SIZES: Record<string, string> = {
+  xs: "1rem",
+  sm: "1.25rem",
+  md: "1.5rem",
+  lg: "2rem",
+  xl: "3.5rem",
+};
+
+export const Loader = ({ size = "md", ...props }: LoaderProps) => (
+  <MantineLoader {...props} size={SIZES[size] ?? size} />
```

Comment:
I'm surprised their types allow to pass any string as size, but should we?

Reply:
Currently all mantine components do. I'm not sure now, but we should keep an eye of this causes issues